src/Transformations.jl

```diff
@@ -1,5 +1,54 @@
 module Transformations
 
-# package code goes here
+## Dependencies ##
+
+using Reexport
+@reexport using LearnBase
+
+import StatsBase:
+    # functions
+    logistic,
+    logit
+
+import Distributions:
+    # probability distributions
+    Distribution,
+    Normal,
+    Bernoulli,
+    NegativeBinomial,
+    Poisson
+
+## Exported types ##
+
+export
+    # static transformations
+    IdentityTransformation,
+    LogTransformation,
+    ExpTransformation,
+    LogisticTransformation,
+    LogitTransformation,
+    ScaleTransformation,
+    ShiftTransformation,
+
+    # stochastic transformations
+    GaussianTransformation,
+    PoissonTransformation,
+    BernoulliTransformation,
+    NegativeBinomialTransformation,
+    GeneralizedLinearTransformation
+
+## Exported functions ##
+# TODO: should these be defined in LearnBase?
+export invert, invert!,
+       get_inverse, isinvertible
```

Comment:
maybe overload `Base.inv` instead of `get_inverse`. I remember reading that a `get` prefix is generally frowned upon in julia